### DISCUSSION TITLE
Need help with integrating MindsDB with MariaDB

### DISCUSSION THREAD

**Discussion by shreeup** [mindsdb/mindsdb#6377] [Get help]
I was trying to work on https://github.com/mindsdb/mindsdb/issues/6221

As part of this defect I created MariaDB in SkqSQL. 
![image](https://github.com/mindsdb/mindsdb/assets/16798480/18bdaf31-2a06-4c46-91bb-c70957bf2a11)

These are tables in Maria DB
![image](https://github.com/mindsdb/mindsdb/assets/16798480/05faf5d5-ef06-475e-ab95-f20690334ab0)

when i try to create integration by executing 
CREATE DATABASE maria_datasource
WITH
  engine = 'mariadb',
  parameters = {
    "host": "dbpwf23068332.sysp0000.db1.skysql.mariadb.com",
    "port": 3306,
    "database": "test",
    "user": "dbpwf23068332",
    "password": "RwDNo9eti6F.SxtI9VsR)0c"
  };
  
  I get an error saying 
  Can't connect to db: 1927 (HY000): Authentication to '@@Xpand-Monitor:node-1' failed: 1, #HY000: [25606] Unsupported encoding: client requested charset 255 (@@Xpand-Monitor:node-1)
  
  Can I get some help on this?

**Comment by ShruAgarwal** [comment]
@shreeup I think here you're trying to connect to MindsDB through **MariaDB SkySQL Editor**. If it is true then you should change the "`host`" value to `"host": "cloud.mindsdb.com"`
> CREATE DATABASE maria_datasource
> WITH
> engine = 'mariadb',
> parameters = {
> "host": "dbpwf23068332.sysp0000.db1.skysql.mariadb.com",

If this doesn't help, then do read their **MariaDB SkySQL Setup [guide here!](https://docs.mindsdb.com/connect/connect-mariadb-skysql)**
